src/app/components/page-summary/index.html

```diff
@@ -18,75 +16,63 @@
     </sky-alert>
   </sky-demo-page-summary>
 
-  <stache-page-anchor>
-    Title and subtitle
-  </stache-page-anchor>
-
+  <sky-demo-page-content sectionHeading="Title and subtitle">
```

Comment:
We need to change the `sky-demo-page-content` sections to `sky-demo-page-properties` sections. The `sky-demo-page-content` sections always appear at the end of the page, which is why UX guidelines are always last. This didn't matter on the button or form docs because those docs don't have demos, but the default behavior of the `sky-demo-page-content` sections messes things up a bit here. Changing these sections to use `sky-demo-page-properties` resolves the placement issue, and since all the sections just include a couple `p` tags, I don't think it messes anything else up. .... We should also bring this up with the full team to make sure there are no issues that I'm overlooking with this workaround and to see if we want devote engineering resources to working on a property for `sky-demo-page-content` that provides flexibility in where to place those sections.